eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java

```diff
@@ -62,55 +63,17 @@
     defaultPhase = LifecyclePhase.PROCESS_SOURCES,
     threadSafe = true
 )
-public final class ProbeMojo extends SafeMojo {
```

Comment:
@maxonfjvipon Maybe it's better to remove one of that mojos and make probing and pulling together in a single mojo? Actually we can use `composition` for it  (by creating two different objects and using it in a single mojo).
Inheritance it is suboptimal solution here, to be honest:
- https://www.yegor256.com/2016/09/13/inheritance-is-procedural.html
- https://en.wikipedia.org/wiki/Composition_over_inheritance

 Josh Bloch's book Effective Java 2nd Edition:
> Item 16: Favor composition over inheritance